astro/src/content/docs/lifecycle/manage-users/account-management/customizing-account-management.mdx

```diff
@@ -82,7 +82,7 @@ Below, we will show you how to make this editable `birthDate` field show up on t
 ### Update Your Theme and Template (Admin Facing)
 
 <Aside type="note">
-For more information on themes please reference the documentation found [here](/docs/customize/look-and-feel/). +
+For more information on themes please reference the documentation found [here](/docs/customize/look-and-feel/).
```

Comment:
I think the reason for the `+` was a line break. So shouldn't this look like this?

```
For more information on themes please reference the documentation found [here](/docs/customize/look-and-feel/).

For more information on custom forms please reference documentation found [here](/feature/user-registration)
```

Or are we intentionally removing the line break?

Reply:
Probably got lost in the shuffle. Fixed.